aos/aos_milestones/IMpg/drive_mtaos.ipynb

```diff
@@ -0,0 +1,356 @@
+{
```

Comment:
<p>The 10 minute time estimate - is this using small, medium or large?</p><p>Does the butler prune-collection command require setting up the stack on the command line?</p><p>How does the OFC get the wavefront measurements? from SAL?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>The WEP was executed by the MTAOS, not from the notebook. The MTAOS runs on an unrestricted mode, so it has access to the full resources available on the server it runs. It was configured to run with 9 processes. </p><p> </p>